scripts/src/javatests/org/oppia/android/scripts/common/BazelClientTest.kt

```diff
@@ -319,6 +319,43 @@ class BazelClientTest {
     assertThat(testTargets).containsExactly("//:FirstTest", "//:SecondTest")
   }
 
+  @Test
+  fun testRetrieveThirdPartyMavenDepsList_forTestBinary_returnsDependenciesList() {
+    testBazelWorkspace.initEmptyWorkspace()
```

Comment:
@BenHenning This test is failing because the query is taking a little longer than 15 seconds - 
```
There was 1 failure:
1) testRetrieveThirdPartyMavenDepsList_forTestBinary_returnsDependenciesList(org.oppia.android.scripts.common.BazelClientTest)
java.lang.IllegalStateException: Expected non-zero exit code (not 7) for command: [bazel, query, deps(deps(//:test_oppia) intersect //third_party/...) intersect @maven//...].
Standard output:

Error output:
$TEST_TMPDIR defined: output root default is '/home/prayutsu/.cache/bazel/_bazel_prayutsu/65c630ff6a7321024a3fd83ac1880b0b/sandbox/linux-sandbox/58/execroot/__main__/_tmp/f65578bdd5e6a06f0954ab44d157a1f1' and max_idle_secs default is '15'.

```

![Screenshot from 2021-07-14 18-13-21](https://user-images.githubusercontent.com/54636525/125623924-07815392-08a0-45f7-91c6-e37909a5ec5a.png)

The default process timeout is configured to be 60 seconds in CommandExecutorImpl() then why the test failed for just 15 seconds?

Reply:
@BenHenning The test if failing even after specifying the --max_idle_secs=60000.
![Screenshot from 2021-07-15 12-29-15](https://user-images.githubusercontent.com/54636525/125743653-fabbe246-d98e-4474-90cd-e78af7058882.png)

I also tried specifying this option after `query` just like other methods but this also failed - 

![Screenshot from 2021-07-15 12-37-17](https://user-images.githubusercontent.com/54636525/125744646-5aee1639-6b6a-428a-9aa7-a338471e5665.png)

Also, you asked to update BazelClient to use a default idle limit of 60 seconds but I couldn't see any code relevant to that.
Can you please elaborate how to do that?

Reply:
Fixed.